test/test_ecc.c

```diff
@@ -1411,7 +1411,7 @@ int test_ec_load_key(void* data)
     params[0] = OSSL_PARAM_construct_end();
 
     PRINT_MSG("Open ECC private key");
-    ctx = OSSL_STORE_open_ex("./certs/ecc-key.pem", wpLibCtx, NULL, NULL, NULL,
+    ctx = OSSL_STORE_open_ex(CERTS_DIR"ecc-key.pem", wpLibCtx, NULL, NULL, NULL,
```

Comment:
see below note.  should be `OSSL_STORE_open_ex(CERTS_DIR "/ecc-key.pem", ...)`.  note also it's fine to put whitespace between `CERTS_DIR` and `"/ecc-key.pem"` -- makes for clearer code, and is ignored by the compiler.